File: fix_sales.js
import fs from 'fs';
let content = fs.readFileSync('src/components/SalesModule.tsx', 'utf8');
content = content.replace('                  }}\n                  disabled={isSaving}\n}}\n                disabled={isSaving}', '                  }}\n                  disabled={isSaving}');
fs.writeFileSync('src/components/SalesModule.tsx', content, 'utf8');
console.log('Fixed SalesModule');

File: fix_sales.py
with open('/app/applet/src/components/SalesModule.tsx', 'r') as f:
    lines = f.readlines()

# Remove the last 3 lines
lines = lines[:-3]

with open('/app/applet/src/components/SalesModule.tsx', 'w') as f:
    f.writelines(lines)
    f.write("      </div>\n")
    f.write("    </div>\n")
    f.write("  );\n")
    f.write("};\n")
